### PR TITLE
Restore #2703 changes

### PR DESCRIPTION
**Pull request by hryhoriivoloshyn**
This pull request is created to restore changes from https://github.com/ita-social-projects/EPlast/pull/2703

For some reason changes from that pull request were not truly merged into master.

### CODE REVIEW

**Review by tvardero — APPROVED** [review]

File: EPlast/EPlast.BLL/Services/UserProfiles/UserProfileAccessService.cs [hunk 1]
```diff
@@ -20,64 +21,98 @@ public UserProfileAccessService(IUserService userService, UserManager<DatabaseEn
...
             {
                 return true;
             }
             return
-                ((roles.Contains(Roles.OkrugaHead)) && await _userService.IsUserInSameCellAsync(currentUser, focusUser, CellType.Region)) ||
-                ((roles.Contains(Roles.CityHead)) && await _userService.IsUserInSameCellAsync(currentUser, focusUser, CellType.City)) ||
-                ((roles.Contains(Roles.KurinHead)) && await _userService.IsUserInSameCellAsync(currentUser, focusUser, CellType.Club));
+                ((currentUserRoles.Contains(Roles.OkrugaHead)) && await _userService.IsUserInSameCellAsync(currentUser, focusUser, CellType.Region)) ||
+                ((currentUserRoles.Contains(Roles.CityHead)) && await _userService.IsUserInSameCellAsync(currentUser, focusUser, CellType.City)) ||
+                ((currentUserRoles.Contains(Roles.KurinHead)) && await _userService.IsUserInSameCellAsync(currentUser, focusUser, CellType.Club));
```
Comment (tvardero) [inline EPlast/EPlast.BLL/Services/UserProfiles/UserProfileAccessService.cs:56]: зайві дужки

**Review by Mykola123-NMT — APPROVED** [review]

**Review by vsynieok — APPROVED** [review]

**Review by tvardero — CHANGES_REQUESTED** [review]
йой а чого так багато коммітів

File: EPlast/EPlast.BLL/Services/UserProfiles/UserProfileAccessService.cs (hunk 1 above)
Comment (hryhoriivoloshyn) [reply EPlast/EPlast.BLL/Services/UserProfiles/UserProfileAccessService.cs]: Got it

**Comment by hryhoriivoloshyn** [comment]
> йой а чого так багато коммітів

Here is a normal version of this pull request, I tried fixing this branch with interactive rebase but got confusing results, so I just created branch based on master and squashed changes I needed into that branch, should be much better.
https://github.com/ita-social-projects/EPlast/pull/2785

**Pull request closed without merging**